OODebug/src/debugger/JavaDebugger.cpp

```diff
@@ -230,8 +245,21 @@ Interaction::CommandResult* JavaDebugger::probe(OOVisualization::VStatementItemL
 
 	if (arguments[0] == "-")
 	{
-		removeObserverOverlaysAt(observedNode, vItem);
-		removeBreakpointAt(observedNode);
+		qint64 idToRemove = -1;
+		if (arguments.size() > 1)
+			idToRemove = arguments[1].toInt();
+		for (auto it = probes_.find(observedNode); it != probes_.end() && it.key() == observedNode;)
+		{
+			if (idToRemove < 1 || it->breakpointId_ == idToRemove)
```

Comment:
So ids -1 and 0 have a special meaning? We might want to document this at the point we define the breakpoint type.